src/test/java/ca/nines/ise/writer/XMLWriterInlinesTest.java

```diff
@@ -0,0 +1,99 @@
+package ca.nines.ise.writer;
+
+import ca.nines.ise.dom.DOM;
+import nu.xom.*;
+import org.junit.*;
+import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.is;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+import org.junit.runners.Parameterized.Parameter;
+import java.util.regex.Pattern;
+import java.util.List;
+import java.util.Arrays;
+
+
+@RunWith(Parameterized.class)
+public class XMLWriterInlinesTest extends XMLWriterTestBase {
+
+    @Parameters
+    public static List<Object[]> inlineTags() {
```

Comment:
Agreed. I've tried very hard to keep data and code separate, especially for cases where the data can be reused for different purposes.